### PR TITLE
Fix/nwp time

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

Change nwp target_time to time

Fixes #

## How Has This Been Tested?

_Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.
Please also list any relevant details for your test configuration_

- [ ] No
- [x] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by jacobbieker** [comment]
Oops! Sorry for marking as ready, mixed this up with mine!

**Comment by peterdudfield** [comment]
> Oops! Sorry for marking as ready, mixed this up with mine!

no worries, it is infact ready

File: nowcasting_dataloader/data_sources/satellite/satellite_model.py [hunk 1]
```diff
@@ -113,6 +112,15 @@ def get_datetime_index(self) -> Array:
     @staticmethod
     def from_xr_dataset(xr_dataset: xr.Dataset):
         """Change xr dataset to model."""
+
+        # channels = xr_dataset.channels
+        #
```
Comment (jacobbieker) [inline nowcasting_dataloader/data_sources/satellite/satellite_model.py:117]: Can we just delete this?

**Review by jacobbieker — APPROVED** [review]
LGTM! Could you just remove the commented out code?

File: nowcasting_dataloader/data_sources/satellite/satellite_model.py (hunk 1 above)
Comment (peterdudfield) [reply nowcasting_dataloader/data_sources/satellite/satellite_model.py:117]: oh yea, sorry, that is for a later PR

**Pull request merged**